src/main/webapp/index.jsp

```diff
@@ -46,6 +46,7 @@
             create or join a conversation.</li>
         <li>View the <a href="/about.jsp">about</a> page to learn more about the
             project.</li>
+        <li>View the <a href="/admin">admin</a> page for app-related statistics!</li>
```

Comment:
In your final product, you might consider hiding this option from non admin users.  But probably better not to do that now.